README.md

```diff
@@ -1,7 +1,7 @@
 # powsybl-diagram-viewer
 
-Typescript library to integrate a powsybl svg diagram in a javascript project. The library is built with the Parcel bundler.  
-Node from v16.15.0 is required to build with Parcel.
+Typescript library to integrate a powsybl svg diagram in a javascript project. The library is built with the Vite bundler.
+Node from v16.15.0 is required to build with Vite.
```

Comment:
It's the same node version for Vite ?

Reply:
And don't forget to change the tsconfig.json file

Reply:
We could get inspiration from the exemple project for package.json and tsconfig.json : 
https://stackblitz.com/edit/vitejs-vite-38rubf?file=tsconfig.json&terminal=dev

Reply:
we can add this check in `package.json`:
```jsonc
{
    //...
    engine: {
        "node": ">=18",
        "npm": ">=10"
    },
    //...
}
```

Reply:
TL don't want this, I already tried to add it

Reply:
@flomillot 
@Tristan-WorkGH 
several points in those comments :
- update to node 18 -> I agree, updating README
- updating tsconfig.json -> why ? It's not the point of this PR
- updating package.json with "engine (...)" -> I agree, it's already there in commons-ui actually

Reply:
For instance :
https://github.com/powsybl/powsybl-diagram-viewer/blob/3bf3c2747348f12314c842a3d56650d53e2387a1/tsconfig.json#L3-L4

Reply:
updated to es6 for both to match gridstudy versions